plugin/templar.vim

```diff
@@ -12,4 +12,4 @@ augroup Templar
 augroup END
 
 
-command! -nargs=1 TemplarRegister lua require'templar'.register(<f-args>)
+command! -nargs=* TemplarRegister lua require'templar'.register(<f-args>)
```

Comment:
I think we can have a `+` here instead of the `*`, because we want at least one argument.

Reply:
Yeah that actually makes sense :)